course_info/templates/course_info/widget.html

```diff
@@ -4,11 +4,21 @@
     <base target="_blank">
 {% endblock head_extra %}
 {% block extra_js %}
-  <script src="{% static 'js/iframeResizer.contentWindow.min.js' %}"></script>
+  {% if referer %}
+    <script>
+    $(document).ready(function() {
+      h = document.body.offsetHeight;
+      data = {'message': 'ready_for_resize', 'height': h};
+      window.parent.postMessage(data, '{{ referer }}');
+      console.log('child frame ready_for_resize to '+ h);
```

Comment:
Can this be cleaned up after testing?

Reply:
yeah - we can always re-enable it later if needed